pkg/node/node.go

```diff
@@ -286,6 +287,21 @@ func (n Node) TaintSpotItn(nodeName string, eventID string) error {
 	return addTaint(k8sNode, n, SpotInterruptionTaint, eventID, corev1.TaintEffectNoSchedule)
 }
 
+// LogPods logs all the pod names on a node
+func (n Node) LogPods(nodeName string) error {
```

Comment:
Can you add a unit test or two for this within `node_test.go` ?

Reply:
```
func TestLogPods(t *testing.T) {
	resetFlagsForTest()

	client := fake.NewSimpleClientset(
		&v1.Pod{
			ObjectMeta: metav1.ObjectMeta{
				Name: "myPod",
				Labels: map[string]string{
					"spec.nodeName": nodeName,
				},
			},
		},
		&v1.Node{
			ObjectMeta: metav1.ObjectMeta{
				Name: nodeName,
			},
		},
	)

	tNode := getNode(t, getDrainHelper(client))
	err := tNode.LogPods(nodeName)
	h.Ok(t, err)
}
```

Reply:
Yes, test added. Thanks for your code snippet.